### ISSUE TITLE
nb-datepicker icons missing for navigations

### ISSUE THREAD

**Issue by maihannijat** [akveo/nebular#1355]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [ x] bug report
* [ ] feature request

### Issue description

**Current behavior:**
Next and Previous navigation icons are missing.

**Expected behavior:**
Next and Previous navigation icons to go to next or previous months.

**Steps to reproduce:**
- Use nebular 4
- Use eva-icons instead of nebular-icons
- Remove nebular-icons package and style from Angular.json

**Related code:**
```
<i _ngcontent-c43="" class="nb-arrow-left" ng-reflect-ng-class="[object Object]"></i>
<nb-calendar-navigation _ngcontent-c43="" _nghost-c45="" ng-reflect-date="Thu Apr 04 2019 09:23:13 GMT-0">
<button _ngcontent-c45="" nbbutton="" _nghost-c41="" tabindex="0"> Apr 2019 </button></nb-calendar-navigation>
<i _ngcontent-c43="" class="nb-arrow-right" ng-reflect-ng-class="[object Object]"></i>
```

**Comment by lebaudantoine** [comment]
I am having the same issue

**Comment by nnixaa** [comment]
fixed in 4.0
